components/expenses/ExpensesFilters.js

```diff
@@ -41,7 +42,7 @@ const ExpensesFilters = ({ collective, filters, onChange }) => {
   });
 
   return (
-    <Flex flexWrap="wrap">
+    <Flex flexWrap={wrap ? 'wrap' : 'nowrap'}>
```

Comment:
@kewitz This breaks the page on mobile, why not wrap?

**Before**
![image](https://user-images.githubusercontent.com/1556356/166923483-467eb17f-b601-4a3c-b633-b67457935a03.png)

**After**
![image](https://user-images.githubusercontent.com/1556356/166923585-c634c644-8c24-41a4-bcdf-03b658704440.png)

Reply:
Ouch. Sorry, I'll fix this.
The problem with wrapping this is that it will break the line in the desktop and display the Order field with a full width on a new line.